src/lib/isBtcAddress.js

```diff
@@ -1,9 +1,14 @@
 import assertString from './util/assertString';
 
 // supports Bech32 addresses
-const btc = /^(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,39}$/;
+const bech32 = /^(bc1)[a-z0-9]{25,39}$/;
+const base58 = /^(1|3)[A-HJ-NP-Za-km-z1-9]{25,39}$/;
```

Comment:
We have `isBase58` validator, how about re-using it here?

Reply:
I only have an issue with the reuse since:

- `base58` will only be needed on `A-HJ-NP-Za-km-z1-9` hence we will need to add `(1|3)` in front of it and check that `base58 has 25-39 chars`. 
Based on the above, I realized it will be hard to read and maintain the code

Reply:
I see, makes sense. I'm good then.